src/VerletList.hpp

```diff
@@ -85,11 +87,41 @@ namespace espressopp {
     /** Set the number of times the Verlet list has been rebuilt */
     void setBuilds(int _builds) { builds = _builds; }
 
+    void resetTimers();
+
+    void loadTimers(real* t);
+
     /** Register this class so it can be used from Python. */
     static void registerPython();
 
   protected:
 
+    std::vector<real> c_x,c_y,c_z;
+    std::vector<Real3D> c_pos;
+    std::vector<Particle*> c_p;
+    std::vector<size_t> c_id, c_type;
+
+    inline void rebuildUsingBuffers(bool useExList, bool useSOA)
+    {
+      if(useExList) {
```

Comment:
Does the template here speed up?

Reply:
Yes though I'm not sure by how much. The template allows the compiler to skip compilation of code under `if(USE_EXCLUSION_LIST)` when the parameter is set to 0. I can confirm this with the source code / assembly viewer on VTune which shows that these source code sections do not have corresponding entries in the assembly.